cmd/geth/chaincmd.go

```diff
@@ -50,6 +50,8 @@ var (
 		ArgsUsage: "<genesisPath>",
 		Flags: flags.Merge([]cli.Flag{
 			utils.CachePreimagesFlag,
+			utils.OverrideCancun,
+			utils.OverrideVerkle,
```

Comment:
Looks like the overrides were added to the `geth init` command. Don't really think we need the same for the optimism fork overrides though, anyone doing a `geth init` can just set the chain config vars in the `genesis.json` before the `geth init`